tools/esdlcmd/esdl-publish.cpp

```diff
@@ -615,7 +599,7 @@ class EsdlDeleteESDLDefCmd : public EsdlPublishCmdCommon
                     optESDLService.set(arg);
                     break;
                 case 1:
-                    optVersionStr.set(arg);
+                    optDefinitionVersionStr.set(arg);
```

Comment:
Especially when it looks like we turn it into an ID before sending it to ESP.

If we make that change we may not need the variable anymore, but I also wonder if the name "PublishedVersion" is more descriptive... meaning the particular version of the ESDLService that was published for "myserv.3"